jekyll/assets/js/main.js

```diff
@@ -261,10 +258,81 @@ $( document ).ready(function() {
     $("nav.sidebar").toggleClass("open");
   });
 
+  var tooltip = document.querySelector(".server-version-popover");
```

Comment:
Where is this coming from? Are we reusing something that was already there?

Reply:
yeah I'm reusing the popover for the top badges. just moved the line where it occurs. 

I could rename it generalize it.

Reply:
That would be great! thank you

Reply:
done!